network/no_ingress_conn_alert.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package network
+
+import (
+	"errors"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/constants"
+)
+
+// ErrNoIngressConnections denotes that no node is connected to this validator.
+var ErrNoIngressConnections = errors.New("no ingress connections")
+
+type ingressConnectionCounter interface {
+	IngressConnCount() int
+}
+
+type validatorRetriever interface {
+	GetValidator(subnetID ids.ID, nodeID ids.NodeID) (*validators.Validator, bool)
+}
+
+type noIngressConnAlert struct {
+	selfID             ids.NodeID
+	ingressConnections ingressConnectionCounter
+	validators         validatorRetriever
+}
+
+func ingressConnResult(n int, areWeValidator bool) map[string]interface{} {
+	return map[string]interface{}{"ingressConnectionCount": n, "primary network validator": areWeValidator}
```

Comment:
It feels odd that "one of these has spaces" and the other is "camelCase".

Reply:
Fixed [here](https://github.com/ava-labs/avalanchego/pull/3719/commits/9c9ca56e1ad115a5e5ba025a4df288e978e4ec12#diff-fd96d1e5d5ab19a672f4a33d6ab4a3698cca6103c509465354b2f07377f0dcc1L47).